src/main/java/nl/workingtalent/wtacademy/user/UserController.java

```diff
@@ -21,76 +25,163 @@ public class UserController {
 
 	// READ
 	@RequestMapping("user/all")
-	public List<User> allUsers() {
-		return service.findAllUsers();
+	public ResponseDto findAllUsers() {
+		List<User> users = service.findAllUsers();
+		Stream<ReadUserDto> readUserDtoStream = getUsers(users);
+
+		if (users.isEmpty()) {
+			ResponseDto responseDto = new ResponseDto(false, null, null, "No users found.");
+			return responseDto;
+		}
```

Comment:
De stream kan na de check of er users zijn

Reply:
Goeie!